transforms/ebml_schema2markdown4rfc.xsl

```diff
@@ -54,9 +54,9 @@
       <xsl:value-of select="@range"/>
       <xsl:text>&#xa;&#xa;</xsl:text>
     </xsl:if>
-    <xsl:if test="@size">
-      <xsl:text>size:&#xa;: </xsl:text>
-      <xsl:value-of select="@size"/>
+    <xsl:if test="@length">
+      <xsl:text>length:&#xa;: </xsl:text>
+      <xsl:value-of select="@length"/>
```

Comment:
@mcr We should include this before we submit for more reviewing (unless it has been done already). Otherwise all the length fields are not mentioned in the spec, which doesn't include the non normative XML file.